### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by artsiom-tsaryonau**

### CODE REVIEW

**Comment by AShynkevich** [comment]
Please add the README with starting description

File: homework5/src/main/resources/create_tables.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CREATE TABLE USERS (
+    id INT PRIMARY KEY,
+    name VARCHAR(128)
+);
```
Comment (AShynkevich) [inline homework5/src/main/resources/create_tables.sql]: Add new line

File: homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                        Field field = expectedObject.getDeclaredField(columnName.toLowerCase());
+                        field.setAccessible(true);
+                        field.set(toReturn, resultSetValue);
+                    } catch (NoSuchFieldException e) {
+                        LOGGER.warn("Object [{}] does not contain field [{}].", expectedObject,
+                                columnName.toLowerCase());
+                    }
+                }
+            }
+        } catch (SQLException | IllegalAccessException | InstantiationException e) {
```
Comment (AShynkevich) [inline homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java]: I think, it makes sense to swap for one Exception

File: homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        LOGGER.debug("Starting session");
+        DatabaseConnectivity.INSTANCE.createSession();
+    }
+
+    public void closeSession() {
+        LOGGER.debug("Closing session");
+        DatabaseConnectivity.INSTANCE.destroySession();
+    }
+
+    public <T> T readForObject(String templateId, String... args) {
```
Comment (AShynkevich) [inline homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java:29]: Is it possible to refactor this method? (separate by several small)

File: homework5/src/main/java/com/epam/lab/mentoring/orm/OrmConstants.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package com.epam.lab.mentoring.orm;
+
+public final class OrmConstants {
+
+    final public static String PROPERTIES_FILE = "database.properties";
+    final public static String DB_URL = "database.url";
+    final public static String DB_USERNAME = "database.username";
+    final public static String DB_PWD = "database.password";
+    final public static String DB_DRIVER = "database.driver";
+
```
Comment (AShynkevich) [inline homework5/src/main/java/com/epam/lab/mentoring/orm/OrmConstants.java]: remove line

File: homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java (hunk 3 above)
Comment (artsiom-tsaryonau) [reply homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java:29]: Moved object fields population to a separate method

File: homework5/src/main/resources/create_tables.sql (hunk 1 above)
Comment (artsiom-tsaryonau) [reply homework5/src/main/resources/create_tables.sql]: fixed

File: homework5/src/main/java/com/epam/lab/mentoring/orm/OrmConstants.java (hunk 4 above)
Comment (artsiom-tsaryonau) [reply homework5/src/main/java/com/epam/lab/mentoring/orm/OrmConstants.java]: fixed

File: homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java (hunk 2 above)
Comment (artsiom-tsaryonau) [reply homework5/src/main/java/com/epam/lab/mentoring/orm/database/DatabaseSession.java]: fixed

**Pull request merged**